.github/pull_request_template.md

```diff
@@ -1,9 +1,12 @@
-**Description:** <Describe what has changed.>
 <!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
 Ex. Adding a feature - Explain what this achieves.-->
+Description:
 
-**Link to tracking Issue:** <Issue number if applicable>
+ <!--Issue number if applicable-->
+Fixes #
```

Comment:
I realize this may end up closing some issues inadvertently if a user doesn't realize the purpose this, but I agree with this approach. I think this is more obvious to users who can just delete the wording, vs. having to add it without being aware.